src/main_app/public/main_routes/inject_routes.py

```
@@ -222,46 +238,88 @@ def inject_get(self, source: str, target: str) -> str:
+        src_langs_sorted = []
+        src_langs = set()
+
+        for entry in new_translations.values():
+            if isinstance(entry, dict):
+                src_langs.update(entry.keys())
+
+        src_langs_sorted = sorted(src_langs)
+        return src_langs_sorted
+
+    def inject_demo(self) -> str:
+        dir = Path(__file__).parent.parent.parent.parent
```

Ruff (0.16.1)

[error] Variable `dir` is shadowing a Python builtin

(A001)